schema/Readme.md

```diff
@@ -1104,7 +1092,7 @@ The schema can require certain environment expectations to be met. The purpose o
 ```groovy
 datasource pg {
   provider = "postgres"
-  url      = env("POSTGRES_URL")
+  url      = env("DATABASE_URL")
```

Comment:
Unrelated change to standardize to `DATABASE_URL`?

Reply:
Yes. Can move to a different PR if you prefer.

Reply:
Adding to title is enough